src/main/webapp/project/NewProject.js

```diff
@@ -496,6 +528,17 @@ class NewProject extends Component {
           errors: this.state.showErrorIntCohorts,
           origin: 'newProject'
         }),
+        Security({
+          title: "Security",
+          step: 3,
```

Comment:
Ah ... this looks like the answer to my previous question about checking against `3`. The connection between here and there is non-obvious and definitely needs some comments about it.

Reply:
As matter a fact, this has nothing to do with the number 3 you saw above. I checked the entire code trying to understand its usage (if any) and couldn't find one... Since this code had some simplification on its usage but at the same time I tried to keep at minimum too much changes, it seems that this is old code now unused. 
I'll keep an eye to clean up this type of orphan code in the rest of the project as I continue refactoring stuff.

Reply:
Nice ... really glad I actually found something :-)